{
  "_from": "eslint-plugin-new-with-error@^2.0.0",
  "_id": "eslint-plugin-new-with-error@2.0.0",
  "_inBundle": false,
  "_integrity": "sha512-8jty4XD7/snyhyClvXaiy0V7fjFiJD0Vw9U4+ntXjcgqhgJjfN1xFMZDBrOObTfHLGCbyprZP2Sl30gjPXRcLw==",
  "_location": "/eslint-plugin-new-with-error",
  "_phantomChildren": {},
  "_requested": {
    "type": "range",
    "registry": true,
    "raw": "eslint-plugin-new-with-error@^2.0.0",
    "name": "eslint-plugin-new-with-error",
    "escapedName": "eslint-plugin-new-with-error",
    "rawSpec": "^2.0.0",
    "saveSpec": null,
    "fetchSpec": "^2.0.0"
  },
  "_requiredBy": [
    "/@websublime/eslint-config"
  ],
  "_resolved": "https://registry.npmjs.org/eslint-plugin-new-with-error/-/eslint-plugin-new-with-error-2.0.0.tgz",
  "_shasum": "ff84db8b25365945a6a2fc694c3e6d93b8f10ad8",
  "_spec": "eslint-plugin-new-with-error@^2.0.0",
  "_where": "C:\\Users\\DannyMax\\IdeaProjects\\django\\CodingMosh\\Me2U\\node_modules\\@websublime\\eslint-config",
  "author": {
    "name": "Rich Trott",
    "email": "rtrott@gmail.com"
  },
  "bugs": {
    "url": "https://github.com/Trott/eslint-plugin-new-with-error/issues"
  },
  "bundleDependencies": false,
  "dependencies": {},
  "deprecated": false,
  "description": "Require `throw new Error()` and not `throw Error()`",
  "devDependencies": {
    "eslint": "^5.0.1"
  },
  "directories": {
    "doc": "docs",
    "test": "tests"
  },
  "homepage": "https://github.com/Trott/eslint-plugin-new-with-error#readme",
  "keywords": [
    "eslint",
    "eslint-plugin"
  ],
  "license": "MIT",
  "name": "eslint-plugin-new-with-error",
  "peerDependencies": {
    "eslint": ">=2.13.1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/Trott/eslint-plugin-new-with-error.git"
  },
  "scripts": {
    "test": "node tests/lib/rules/new-with-error.js"
  },
  "version": "2.0.0"
}
